ddl/reorg.go

```diff
@@ -152,6 +154,7 @@ func (w *worker) runReorgJob(t *meta.Meta, reorgInfo *reorgInfo, lease time.Dura
 		rowCount, doneHandle := w.reorgCtx.getRowCountAndHandle()
 		// Update a job's RowCount.
 		job.SetRowCount(rowCount)
+		updateAddIndexProgress(w, tblInfo, rowCount)
```

Comment:
Please consider updating the progress to 100 when the DDL job is done. Otherwise, if the table count estimation is larger than the real one, users may see a finished DDL job from the client, but the progress metric is less than 100... That can be confusing.